Source/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Lib.Harmony" version="2.0.0.8" targetFramework="net472" />
+  <package id="Lib.Harmony" version="2.0.1" targetFramework="net471" />
+  <package id="UnlimitedHugs.Rimworld.HugsLib" version="7.2.1" targetFramework="net472" requireReinstallation="true" />
```

Comment:
hugs are needed for what?

Reply:
PrisonerComp long ticks (interval of 500 to 2000 ticks) going to be used monitor mood for incidents, escape tick and jobs tweaks.